packages/editor/src/store/actions.js

```diff
@@ -199,9 +199,9 @@ export const savePost =
 
 		let error = false;
 		try {
-			error = await applyFilters(
-				'editor.__unstablePreSavePost',
-				Promise.resolve( false ),
+			edits = await applyFilters(
+				'editor.PreSavePost',
```

Comment:
Most examples in the codebase are lower case first letter:
```suggestion
				'editor.preSavePost',
```

Reply:
OK, I'll change that to lowercase. I've been looking at prior art for editor filters, and there are names like `editor.BlockListBlock`, `editor.BlockEdit` or `editor.MediaUpload`, but these turn out to be exceptions: they are filters that wrap React components of the same name. By default filters should start with lowercase.